webview/src/shared/components/slider/MinMaxSlider.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FormEvent } from 'react'
+import styles from './styles.module.scss'
+
+interface MinMaxSliderProps {
+  minimum?: number
+  maximum: number
+  step?: number
+  defaultValue: number
+  label: string
+  onChange: (newValue: number) => void
+}
+
+export const MinMaxSlider: React.FC<MinMaxSliderProps> = ({
```

Comment:
Not sure if there is anything that can be done about it but the slider jumps sometimes when the plot rows increase: 

https://user-images.githubusercontent.com/43496356/223739261-93d346ee-8f54-4a8a-addd-d407a4257499.mov

Reply:
The way to solving that would be to make sure the last section is at least `100vh` which does not always look good, especially when it's empty or have very few plots. I'm wondering though if we shouldn't have one width slider for all plots instead of resizing per section.